xtab/org.eclipse.birt.report.item.crosstab.core/src/org/eclipse/birt/report/item/crosstab/core/util/CrosstabUtil.java

```diff
@@ -1116,6 +1116,10 @@ public static boolean validateBinding( ComputedColumnHandle column, String colum
 		{
 			columnName = "[^\\]]+";
 		}
+		else
+		{
+			columnName = processSpecialCharacters( columnName );
```

Comment:
Validation should not change column name, and it should just check invalid or not.

Reply:
fix code to just treat column name as string literal and remove added logic